### DISCUSSION TITLE
Sound on Pro 4 and later

### DISCUSSION THREAD

**Discussion by EdCha** [linux-surface/linux-surface#622] [Q&A]
Hello 
Could someone confirm that sound is actually working on Surface Pro4 and later ?
So far I have little success and solutions over various forums seem to be either out of date or poorly documented.

Many thanks
Ed

**Comment by SamLukeYes** [comment]
Running Arch Linux with GNOME on Surface Pro 6, I can confirm the sound system is fully working out of box.
